apps/system/js/browser_mixin.js

```diff
@@ -57,6 +57,25 @@
     },
 
     /**
+     * This function will be called by window managers while top most app
+     * window is changed to notify nfc module in gecko.
+     */
+    setNFCFocus: function(enable) {
+      if (!this.browser || !this.browser.element ||
+          this._nfcActive === enable ||
```

Comment:
I kinda wish this case was handled by the platform (ie. calling setNFCFocus(true) twice being a no-op the second time).